docs/quickstart.md

```diff
@@ -453,6 +453,36 @@ Out[9]: 210.99245283018868
 ```
 After subtracting the amount of time required to create an array copy from the given times, we can conclude that the pyccelised function is approximately 210 times faster than the original Python function.
 
+### Interactive Usage with `lambdify`
+
+While Pyccel is usually used to accelerate Python code, it is also possible to accelerate other expressions. The Pyccel library provides the `lambdify` Python function. This function is similar to SymPy's [`lambdify`](https://docs.sympy.org/latest/modules/utilities/lambdify.html) function, given a SymPy expression `f` and type annotations, `lambdify` returns a "pyccelised" function `f_fast` that can be used in the same Python session.
+For example:
+```python
+import numpy as np
+import sympy as sp
+from pyccel import lambdify
+
+x = sp.Symbols('x')
+expr = x**2 + x*5
+f = lambdify(expr, {x : 'float'})
+print(f(3.0))
+
+expr2 = x-x
+f2 = lambdify(expr, {x : 'float'}, float)
+print(f2(3.0))
+
+expr = x**2 + x*5 + 4.5
+f3 = lambdify(expr, {x : 'T'}, float, {'T': ['float', 'float[:]', 'float[:,:]']})
```

Comment:
Here the return statement is ignored in the case of an input array. I feel that this is a bit confusing.

Reply:
Oops. I meant to put `T`

Reply:
Actually `'T'` doesn't seem to be possible. I have opened an issue https://github.com/pyccel/pyccel/issues/1835 and corrected the example: [bc3d3f9](https://github.com/pyccel/pyccel/pull/1829/commits/bc3d3f94068cd93fc81532193196ea2843b1d960)